package/libsolv.spec.in

```diff
@@ -15,122 +15,97 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
-Name:           libsolv
-Version:        @VERSION@
-Release:        0
-Url:            https://github.com/openSUSE/libsolv
-Source:         libsolv-%{version}.tar.bz2
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-
-%bcond_without enable_static
-%bcond_without disable_shared
-%bcond_without perl_binding
-%bcond_without python_binding
-%bcond_without python3_binding
-%bcond_without ruby_binding
-%bcond_with zypp
 
+%define libname libsolv@LIBSOLV_SOVERSION@
 %if 0%{?leap_version} >= 420300 || 0%{?sle_version} >= 120300 || 0%{?suse_version} >= 1330 || !0%{?suse_version}
 %bcond_without bz2
 %bcond_without xz
 %else
 %bcond_with bz2
 %bcond_with xz
 %endif
-
 %if 0%{?fedora} || 0%{?rhel} >= 7 || 0%{?mageia} >= 6 || 0%{?suse_version} >= 1330
 %bcond_without richdeps
 %else
 %bcond_with richdeps
 %endif
-
-%if 0%{?mandriva_version}
-# force this version on mandriva
-BuildRequires:  libneon0.26-devel
+%bcond_without enable_static
+%bcond_without disable_shared
+%bcond_without perl
+%bcond_without python2
+%bcond_without python3
+%bcond_without ruby
+%bcond_with zypp
+Name:           libsolv
+Version:        @VERSION@
+Release:        0
+Summary:        Package dependency solver using a satisfiability algorithm
+License:        BSD-3-Clause
+Group:          Development/Libraries/C and C++
+Url:            https://github.com/openSUSE/libsolv
+Source:         libsolv-%{version}.tar.bz2
+BuildRequires:  cmake
+BuildRequires:  fdupes
```

Comment:
`fdupes` is not available in base RHEL/CentOS. Please restore the conditional.

At least, use `%if 0%{?fedora} || 0%{?mageia} || 0%{?suse_version}`